tests/kernel/pipeline_tests.py

```diff
@@ -122,14 +121,15 @@ def test_create_cell(self, mock_default_context, mock_notebook_environment):
 from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
-from datetime import datetime, timedelta
+from datetime import timedelta
+from pytz import timezone
 
 default_args = {
     'owner': 'Datalab',
     'depends_on_past': False,
     'email': ['foo@bar.com'],
-    'start_date': datetime.strptime('Jun 1 2005  1:33PM', '%b %d %Y %I:%M%p'),
-    'end_date': datetime.strptime('Jun 10 2005  1:33PM', '%b %d %Y %I:%M%p'),
+    'start_date': datetime.datetime.strptime('2009-05-05 22:28:15', '%Y-%m-%dT%H:%M:%SZ').replace(tzinfo=timezone('UTC')),
```

Comment:
Looks like you need to add a T and Z to the sample date/time string to match the specified format. Also in end_date here, and on lines 191 and 192.

Reply:
Done; I refactored this around a little bit, but net net is that I've added a T and Z now.